src/index.html

```diff
@@ -152,18 +176,67 @@ <h2 class="modal__title">New post</h2>
         </form>
       </div>
     </div>
+
     <div class="modal" id="preview-modal">
       <div class="modal__content modal__content_type_preview">
         <button
           aria-label="Close button"
           type="button"
           class="modal__close modal__close_type_preview"
         ></button>
+
         <img class="modal__image" src="" alt="" />
         <p class="modal__caption"></p>
       </div>
     </div>
-    <script defer src="./scripts/validation.js"></script>
-    <script defer src="./scripts/index.js"></script>
+
+    <div class="modal" id="avatar-modal">
+      <div class="modal__content">
+        <button
+          aria-label="Close modal"
+          type="button"
+          class="modal__close"
+        ></button>
+
+        <h2 class="modal__title">Edit profile avatar</h2>
+        <form class="modal__form" id="profile-avatar-form" novalidate>
+          <label class="modal__label" for="profile-avatar-form">
```

Comment:
The `for` and `id` of the input must **coincide** to connect them together. Or you don't need `for` if the input is **inside** the `label` (it gets connected automatically) **Please, correct it everywhere in the code**